### PR TITLE
Fix linting for spec

### PR DESCRIPTION
**Pull request by dblythy**

### CODE REVIEW

**Review by davimacedo — APPROVED** [review]

**Comment by dblythy** [comment]
Can we merge this one @davimacedo @mtrezza

**Comment by mtrezza** [comment]
Did you run lint:fix for spec in this PR? Aside from the command there are no file changes.

File: package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
   "scripts": {
     "start": "node index.js",
     "lint": "eslint --cache ./cloud && eslint --cache index.js && eslint --cache ./spec",
-    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js && eslint --cache  --fix./spec",
+    "lint-fix": "eslint --cache --fix ./cloud && eslint --cache --fix index.js && eslint --cache  --fix ./spec",
```
Comment (mtrezza) [inline package.json:20]: There is a double space before --fix?

**Comment by dblythy** [comment]
> Did you run lint:fix for spec in this PR? Aside from the command there are no file changes.

Correct. The issue is without the space, lint-fix does not run for spec, and the last command does nothing.

**Comment by mtrezza** [comment]
So when you run lint:fix in this PR (with the corrected space), there are no fixes necessary in the spec files?

**Comment by dblythy** [comment]
Oh, right, I get what you mean

**Comment by dblythy** [comment]
Nope, all good!

**Review by mtrezza — APPROVED** [review]
LGTM! Thanks for this PR.

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [1.2.0](https://github.com/parse-community/parse-server-example/releases/tag/1.2.0)

**Pull request merged**